src/amb_kf.c

```diff
@@ -544,7 +544,7 @@ void rebase_mean_N(double *mean, const u8 num_sats, const u8 *old_prns, const u8
   u8 new_ref = new_prns[0];
 
   double new_mean[state_dim];
-  s32 index_of_new_ref_in_old = find_index_of_element_in_u8s(num_sats, new_ref, &old_prns[1]);
+  s32 index_of_new_ref_in_old = find_index_of_element_in_u8s(num_sats-1, new_ref, &old_prns[1]);
```

Comment:
I think num_sats - 1 is the length of &old_prns[1], but the mean is length state_dim